### PR TITLE
Improved README.md

### LINKED ISSUE #57: Reimagine structure with samles etc.
As proposed by @ericdrobinson in https://github.com/AdobeXD/typings/issues/54#issuecomment-506982532:
> Repo Structure - New Issue?
> I would like to suggest that the jsconfig.json and sample.js file get moved into a sample or 
> example folder that's separate from the top level. A tsconfig.json file at the top level then makes > the most sense. This will allow a clear separation of TypeScript and JavaScript files while allowing > you to have a tsconfig.json file that's designed specifically for writing the declaration files.
> 
> Thoughts?

### PR DESCRIPTION
**Pull request by pklaschka**
Updated the README.md, improving ease of getting started, providing a screenshot of autocompletion and an explaination of the maintainance structure of the repository
- fixes #57

### CODE REVIEW

**Comment by pklaschka** [comment]
@ericdrobinson Any thoughts? Thank you very much in advance :slightly_smiling_face:

File: README.md [hunk 1]
```diff
@@ -1,8 +1,34 @@
...
 
 ## Download
-
 The best option to download the typings is to download the latest release from <https://github.com/AdobeXD/typings/releases>.
+
+## Getting started
+These Type Declaration files provide your IDE (e.g. Visual Studio Code or WebStorm) with information about the XD API surface, enabling type checking, autocomplete suggestions, and more. To get started, simply copy the following resources from this repo into your project's main directory:
+
+- The jsconfig.json file.
+- The types directory.
```
Comment (ericdrobinson) [inline README.md]: I would suggest inline code formatting for the `jsconfig.json` file and `types` folder. This is slight but it does clearly call attention to the specific targets "at a glance".

File: README.md [hunk 2]
```diff
@@ -1,8 +1,34 @@
-# typings
+# Adobe XD Plugin API typings
 Typings for Adobe XD API Surfaces
+![typings-supported autocompletion features in JetBrains WebStorm](image.png)
 
-Instructions for using these type definitions to get autocompletion features in editors and IDEs can be found in the [repository's wiki](https://github.com/AdobeXD/typings/wiki/Autocompletion-in-editors-and-IDEs).
+Detailed instructions for using these type definitions to get autocompletion features in editors and IDEs can be found in the [repository's wiki](https://github.com/AdobeXD/typings/wiki/Autocompletion-in-editors-and-IDEs).
```
Comment (ericdrobinson) [inline README.md]: I would suggest rewording "type definitions" as "type declarations" as it is more consistent with the underlying TypeScript feature that is used.

File: README.md [hunk 3]
```diff
@@ -1,8 +1,34 @@
...
-
 The best option to download the typings is to download the latest release from <https://github.com/AdobeXD/typings/releases>.
+
+## Getting started
+These Type Declaration files provide your IDE (e.g. Visual Studio Code or WebStorm) with information about the XD API surface, enabling type checking, autocomplete suggestions, and more. To get started, simply copy the following resources from this repo into your project's main directory:
+
+- The jsconfig.json file.
+- The types directory.
+
+Your IDE may require further setup to take full advantage of the types. Please see this [wiki page](https://github.com/AdobeXD/typings/wiki/Autocompletion-in-editors-and-IDEs) for more.
```
Comment (ericdrobinson) [inline README.md]: Consider linking directly to the _Editor-/IDE-specific_ instructions section of that page. See:

    https://github.com/AdobeXD/typings/wiki/Autocompletion-in-editors-and-IDEs#editor--ide-specific-instructions

(The difference is the **#editor--ide-specific-instructions** portion tacked onto the end of the link...)

File: README.md [hunk 4]
```diff
@@ -1,8 +1,34 @@
...
+
+You can also use the typings to use TypeScript for your plugin development. For that, you'll have to copy the `types` folder to your project and include it in `the `paths` of your `tsconfig.json`:
```
Comment (ericdrobinson) [inline README.md]: I would suggest rewording this slightly:

> The Type Declaration files also allow you to use TypeScript instead of JavaScript for your plugin development.

The word "typings" is outdated now and seems to [_actually_ refer to](https://stackoverflow.com/a/41575212/5285364) an old tool that provided access to type declaration files ([`typings`](https://github.com/typings/typings)). It was replaced by Definitely Typed back when TypeScript 2.0 launched.

Comment (ericdrobinson) [inline README.md]: There appears to be an errant back-tick (\`) in this line as well:

    `the `paths` of your `tsconfig.json`

The back-tick at the beginning of this snippet should be removed.

File: README.md [hunk 5]
```diff
@@ -1,8 +1,34 @@
...
+      "types/*"
+    ]
+  }
+}
+```
+
+You may also take a look at the `tsconfig.json` file contained in this repository for reference.
+
+## One last thing
+This, while being a part of the Adobe XD CC GitHub organization, is a community-driven project. This means that all changes get implemented manually and may therefore take some time. It should also be noted that the attempt (to keep the project maintainable in the contributors' free time) is to keep this in sync with the official [plugin API documentation](https://adobexdplatform.com/plugin-docs/).
```
Comment (ericdrobinson) [inline README.md]: Get rid of "CC" in this line. Not only has Adobe dropped the "CC" portion from the 2020 versions of their apps, but it's not in the name of the organization anyway :) You might also simply link to the organization:

>This, while being a part of the [AdobeXD](https://github.com/AdobeXD) GitHub organization, is a community-driven project.

Comment (ericdrobinson) [inline README.md]: Change "attempt" to "goal":

>It should also be noted that the **goal**...

**Comment by pklaschka** [comment]
@ericdrobinson Perfect, thank you for your input. I've now implemented these improvements...

**Comment by pklaschka** [comment]
Alright. I'll wait until tomorrow (10:00 UTC), which then serves as two full work days in American time. If there are no objections until then, I'll go ahead and merge it.
